src/test-jdk14/java/com/fasterxml/jackson/databind/failing/RecordWithJsonNaming3102Test.java

```diff
@@ -1,15 +1,16 @@
 package com.fasterxml.jackson.databind.failing;
 
+import org.junit.Ignore;
 import com.fasterxml.jackson.annotation.JsonCreator;
-
 import com.fasterxml.jackson.databind.BaseMapTest;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.ObjectReader;
 import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
 
-// [databind#3102]: fails on JDK 16 which finally blocks mutation
-// of Record fields.
+//[databind#3102]: fails on JDK 16 which finally blocks mutation
+//of Record fields.
+@Ignore
```

Comment:
only the first of these tests fails - can you just ignore the one that fails?

Reply:
Agreed, and that was the first thing I tried. But the `@Ignore` only seems to have any effect if it's applied to the whole class. I just tried it again and got the same result. Please check my math here because I agree that would be the better solution.

Reply:
So, jackson still uses junit3 as opposed to junit4 (or even junit5) - so that might explain why the Ignore annotation doesn't work at the test level - it works well in junit4 style (where you annotate methods with `@Test`).

Could you move the failing tests to the com.fasterxml.jackson.databind.failing package of test-jdk14? Just the failing tests, not the full test classes. So you can the full class to com.fasterxml.jackson.databind.failing but then remove the tests that work and go back to the original class and remove the broken test(s).

The tests in the 'failing' package should be ignored when you run the build with mvn from command line.